packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardWithValidation.js

```diff
@@ -35,15 +35,29 @@ class LoginCardWithValidation extends React.Component {
     window.removeEventListener('keyup', this.toggleCapsLock);
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  static getDerivedStateFromProps(props, state) {
+    // disableSubmit prop will only be used in a not validated login card
+    if (props.validate) return null;
+    if (props.disableSubmit !== state.form.disableSubmit) {
```

Comment:
could you use destructing here?

I think we can update the state if any of the props is different, not just the `disableSubmit`.
You can use something like this: https://github.com/patternfly/patternfly-react/blob/59ca4e920ba7d6fd06420a02784f0c6bd959083f/packages/patternfly-3/patternfly-react/src/components/DualListSelector/DualListControlled.js#L16

Reply:
I think updating the state if **any** of the props change is be outside the scope of this PR